src/api/java/mezz/jei/api/IRecipeRegistry.java

```diff
@@ -124,10 +124,34 @@
 	 * By default, all smelting recipes from {@link FurnaceRecipes#smeltingList} are already added by JEI.
 	 *
 	 * @since JEI 4.2.7
+	 * @deprecated since JEI 4.3.3. Use {@link #createSmeltingRecipe(List, ItemStack)}
 	 */
+	@Deprecated
 	void addSmeltingRecipe(List<ItemStack> inputs, ItemStack output);
 
 	/**
+	 * Add a new smelting recipe while the game is running.
```

Comment:
Should I add:
`Use {@link #addRecipe(IRecipeWrapper, String)} to add the recipe while the game is running.`
To each method's Doc tags as well?